screencasts/index.html

```diff
@@ -22,12 +22,6 @@
           a free online course that provides you five labs, each with a short video followed
           by a series of exercises where you get to code Rails immediately in your browser.
         </p>
-        <p class="footnote">
-          If you’re interested in the history of Rails, we also still have the screencasts
-          made for <a href="http://media.rubyonrails.org/video/rails_blog_2.mov" title="Video about history of rails 2">Ruby on
```

Comment:
Yeah, I don't have those locally. I think some of them are on Youtube? At least that 0.5 video.